src/actions/attack.cpp

```diff
@@ -742,13 +742,18 @@ class attack
 		int damage_;
 		int xp_;
 
+		const battle_context_unit_stats* stats_;
+
 		unit_info(const map_location& loc, int weapon, unit_map& units);
 		unit& get_unit();
 		bool valid();
 
 		std::string dump();
 	};
 
+	//returns true if the wml aborted the attack
```

Comment:
This documentation doesn't say much about the function.

Both of the places that call this function ignore the return code, which doesn't seem to fit with the documentation.

Reply:
> Both of the places that call this function ignore the return code, which doesn't seem to fit with the documentation

no fire_event(1) doesn't ignore it